tests/cli/clean/test_balsamic_clean.py

```diff
@@ -81,19 +80,37 @@ def test_dry_run(
     assert analysis_to_clean in base_store.analyses_to_clean(pipeline="Balsamic")
 
 
-def test_cleaned_at(cli_runner, clean_context: dict, helpers, caplog):
+def test_cleaned_at_valid(cli_runner, clean_context: dict, caplog):
     """Test command with dry run options"""
     # GIVEN a case on disk that could be deleted
     base_store = clean_context["BalsamicAnalysisAPI"].store
-    analysis_to_clean = base_store.analyses_to_clean(pipeline="balsamic")[0]
-    case_id = analysis_to_clean.family.internal_id
+    case_id = "balsamic_case_clean"
     case_path = clean_context["BalsamicAnalysisAPI"].get_case_path(case_id)
     Path(case_path).mkdir()
     # WHEN dry running with dry run specified
     result = cli_runner.invoke(balsamic_run_dir, [case_id, "-y"], obj=clean_context)
 
     # THEN command should say it would have deleted
     assert result.exit_code == EXIT_SUCCESS
-    assert analysis_to_clean.cleaned_at
-    assert analysis_to_clean not in base_store.analyses_to_clean(pipeline="Balsamic")
+    assert base_store.family("balsamic_case_clean").analyses[0].cleaned_at
     assert not Path(case_path).exists()
+
+
+def test_cleaned_at_invalid(cli_runner, clean_context: dict, caplog):
```

Comment:
```suggestion
def test_cleaned_at_dry_run(cli_runner, clean_context: dict, caplog):
```

Reply:
sorry but I will be keeping the name - this is dry run yes, but every test is a dry run. However this one tests specifically that it doesnt set a running analysis to cleaned